test/tokengenerator_test.go

```diff
@@ -0,0 +1,147 @@
+package testutils
+
+import (
+	"bytes"
+	"crypto/rsa"
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/dgrijalva/jwt-go"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/require"
+	jose "gopkg.in/square/go-jose.v2"
+)
+
+func TestTokenGeneratorKeys(t *testing.T) {
+
+	t.Run("create keys", func(t *testing.T) {
+		tokengenerator := NewTokenGenerator()
+		kid0 := uuid.NewV4().String()
+		key0, err := tokengenerator.CreateKey(kid0)
+		require.NoError(t, err)
+		kid1 := uuid.NewV4().String()
+		key1, err := tokengenerator.CreateKey(kid1)
+		require.NoError(t, err)
```

Comment:
you should also verify that `key0` and `key1` are not `nil`

Reply:
fixed here: 2daa68a3e64dc9b8dbb591341c4a70bffdf0482c